### PR TITLE
Fixing Python 3 incompatibility in graphouse_api.py

### PR DESCRIPTION
**Pull request by deniszh**
graphite api also working with python 3, but `hasattr(path, '__iter__')` will be `True` for strings in python 3!
`type(path) != str` will work for python 2.7 and 3.x btw

### CODE REVIEW

File: src/main/pySources/graphouse_api.py [hunk 1]
```diff
@@ -38,7 +38,14 @@ def __init__(self, path, reqkey='empty', graphouse_url='http://localhost:2005'):
         self.path = None
         self.graphouse_url = graphouse_url
 
-        if hasattr(path, '__iter__'):
+        try:
```
Comment (vvv1559) [inline src/main/pySources/graphouse_api.py]: This `try/catch` block looks like redundant

Comment (deniszh) [reply src/main/pySources/graphouse_api.py]: Oh, sorry, that's should be removed, indeed.

Comment (deniszh) [reply src/main/pySources/graphouse_api.py]: Done

**Comment by vvv1559** [comment]
`This branch has conflicts that must be resolved` =(

**Comment by deniszh** [comment]
WTF... I hope fixed now.

**Pull request merged**